### DISCUSSION TITLE
Scratch-Dev-Team

### DISCUSSION THREAD

**Discussion by Shluffy** [ZI-Youtuber/ZI-Youtuber#7] [Q&A] [answered]
So i reaccepted the invite, and I would like admin. Also, I think maybe it will work best to list my org (@Scratch-Bookmarklets) as a side as this isn't exactly related to developing scratch, and I will probably work on the actual scratch.
Also, sorry about the whole entire thing. And can I retain the right s to the repo?

**Comment by zoraezimran** [comment, marked as answer]
@Shluffy Well Just because it is called devs doesn't always mean developing also used for other things we also work on the actual scratch. Also about that org thing I will put a collabration with your org and my org. Also can you send me an invite to your org pls. And yes you can retain rights to the repos that were mainly designed by you and gosoccerboy and will give you admin access but don't change anything until discuss. Also remember to use team chat for disucssions with org. And no worries it was also a bit of my fault. Thanks

**Reply by Shluffy** [reply]
https://github.com/orgs/Scratch-Bookmarklets/ to accept the invite. I'll update the description or my org

**Reply by Shluffy** [reply]
Billing email is set to my side email though, and it is a free plan. Maybe you want that to be switched to the email that is used for scratch-dev-team

**Reply by zoraezimran** [reply]
@Scratch-Dev-Team  is what you should put in the description by the way

**Reply by zoraezimran** [reply]
> Billing email is set to my side email though, and it is a free plan. Maybe you want that to be switched to the email that is used for scratch-dev-team

It is something you made so I don't mind but I don't even have admin access to the org so I can't swith the email to our main org

**Reply by zoraezimran** [reply]
Also make your visibily in the the scrcth devs team membership public
